modules/installation-three-node-cluster.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-user-infra.adoc [Eventually]
+// * installing/installing_azure/installing-azure-user-infra.adoc [Eventually]
+// * installing/installing_gcp/installing-gcp-user-infra.adoc [Eventually]
+// * installing/installing_gcp/installing-restricted-networks-gcp.adoc [Eventually]
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_aws/installing-restricted-networks-aws.adoc [Eventually]
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc [Eventually]
+// * installing/installing_vsphere/installing-vsphere.adoc [Eventually]
+// * installing/installing_ibm_z/installing-ibm-z.adoc [Eventually]
+
+[id="installation-three-node-cluster_{context}"]
+= Running a three-node cluster
+
+[IMPORTANT]
```

Comment:
Are you able to use the `technology-preview.adoc` file here instead of copy/pasting the important note?